common/reviews/api/node-core-library.api.ts

```diff
@@ -1,3 +1,9 @@
+// @beta
+class Executable {
+  static spawnSync(command: string, args: string[], options?: IExecutableSpawnSyncOptions): child_process.SpawnSyncReturns<string>;
```

Comment:
>spawnSync [](start = 9, length = 9)

Most of the time, people want a string, but if they don't want a string, or if it's an unusual encoding, we should give them an option to return a buffer with an overload.